runtime/tests/evm_fee_preferences.rs

```diff
@@ -345,53 +384,46 @@ fn evm_call_with_fee_preferences_no_asset_should_fail() {
 	let initial_liquidity = 500 * DOLLARS;
 
 	ExtBuilder::default()
-        .initial_balance(initial_balance)
-        .initialise_eth_accounts(vec![cennznet_address.clone()])
-        .stash(initial_balance)
-        .build()
-        .execute_with(|| {
-            // Alice sets up CENNZ <> CPAY liquidity
-            assert_ok!(Cennzx::add_liquidity(
-				Origin::signed(alice()),
-				CENNZ_ASSET_ID,
-				initial_liquidity, // min. liquidity
-				initial_liquidity, // liquidity CENNZ
-				initial_liquidity, // liquidity CPAY
-			));
-
-            let cpay_balance_before = <GenericAsset as MultiCurrency>::free_balance(&cennznet_address, CPAY_ASSET_ID);
-            let cennz_balance_before = <GenericAsset as MultiCurrency>::free_balance(&cennznet_address, CENNZ_ASSET_ID);
-
-            // Create input parameters for call
-            // This abi has an asset_id of 10 which doesn't exist
-            let abi = hex!("ccf39ea9000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000320000000000000000000000001122334455667788991122334455667788990000000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000000051234567890000000000000000000000000000000000000000000000000000000");
-            let input = abi.to_vec();
-            let max_fee_per_gas = U256::from(20000000000000u64);
-            let max_priority_fee_per_gas = U256::from(1000000u64);
-            let access_list: Vec<(H160, Vec<H256>)> = vec![];
-            let config: EvmConfig = CENNZNET_EVM_CONFIG.clone();
-            assert!(<Runtime as pallet_evm::Config>::Runner::call(
+		.initial_balance(initial_balance)
+		.initialise_eth_accounts(vec![cennznet_address.clone()])
+		.stash(initial_balance)
+		.build()
+		.execute_with(|| {
+			setup_liquidity(initial_liquidity);
+			let receiver_eth: H160 = hex!("7a107Fc1794f505Cb351148F529AcCae12fFbcD8").into();
+
+			let cpay_balance_before = <GenericAsset as MultiCurrency>::free_balance(&cennznet_address, CPAY_ASSET_ID);
+			let cennz_balance_before = <GenericAsset as MultiCurrency>::free_balance(&cennznet_address, CENNZ_ASSET_ID);
+
+			// Create input parameters for call with slippage of 0.1%
+			let transfer_amount: u128 = 123;
+			let input: Vec<u8> = encode_transfer_abi(receiver_eth, transfer_amount);
+			let abi = encode_fee_preferences_abi(10, 50, input);
```

Comment:
nit: `abi` not really the right name, maybe `fee_preferences_input` and elsewhere too